features/sanity_check.feature

```diff
@@ -0,0 +1,5 @@
+Feature: Sanity Check
+  Scenario: Check that Smokey is not fundamentally broken
```

Comment:
I like the intent of this test. We could be more specific about what it's actually checking though, e.g.

```suggestion
  Scenario: Check that Smokey can make network requests
```

If we want to add context, we can [use multiple lines under `Feature`](https://cucumber.io/docs/gherkin/reference/#feature) at the top of the file about the intent behind this file.